client/components/date-picker/README.md

```diff
@@ -23,11 +23,13 @@ module.exports = React.createClass( {
 			{
 				title: '1 other post scheduled',
 				date: new Date( '2015-10-15 10:30' ),
-				type: 'schedulled'
+				type: 'schedulled',
```

Comment:
I know it's not a change, but `scheduled` is spelled wrong here. We should probably just fix it.